src/main/scala/strawman/collection/mutable/ListBuffer.scala

```diff
@@ -12,7 +12,7 @@ import scala.Predef.{assert, intWrapper}
 
 /** Concrete collection type: ListBuffer */
 class ListBuffer[A]
-  extends Seq[A]
+  extends GrowableSeq[A]
```

Comment:
> Why do we need to separately say this is growable, buildable

Actually, in the current design `Growable[A]` takes just one type parameter (the type of the elements). If we want to make `Growable[A]` imply `Buildable[A, C]` then we need to add the `C` type parameter to `Growable` and on all its subclasses. A consequence is that this `C` should be supplied by a `XxxOps` trait, which means that we would have a `GrowableIterableOps` branch in the hierarchy that would be refined for `Set`, `Map` and `Seq`. That would be a lot of additional traits :(

Or, could we assume that all mutable collections are `Growable`? (in such a case we could make `mutable.IterableOps` extend `Buildable` and get rid of the `mutable.Iterable` / `mutable.GrowableIterable` distinction)